### ISSUE TITLE
Question - Source of Folder: \Documents\My Games\Skyrim.INI\SKSE 

### ISSUE THREAD

**Issue by z929669** [ModOrganizer2/modorganizer#2006]
## The problem:
I'm just checking if MO (1.5.1rc2) is creating the folder:
```\Documents\My Games\Skyrim.INI\SKSE```

... and if so, what is the rationale? It seems redundant.

## To Reproduce:

1. Upgrade SkyrimSE to latest runtime version (1.6.1170)
2. Install SKSE64 and some of the common mods dependent upon it
3. Launch the game via MO2 (1.5.1rc2)

## Environment:
* Mod Organizer Version that exhibits the issue: 1.5.1rc2
* Last Mod Organizer Version that did not exhibit the issue (if applicable): ???
* Desktop OS/version used to run Mod Organizer: Windows 11 23H2 (build: 22631.3296)

## Details:
See [forum post](https://stepmodifications.org/forum/topic/19517-sse-161170-or-mo2-now-making-skyrimini-folder/?do=findComment&comment=278206) for testing results and assumptions.

## Link to Mod Organizer logs:
Right click on the log window at the bottom of MO2, set log level to debug, restart Mo2 and reproduce the problem.
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your latest USVFS log, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by flipdeezy** [comment]
ditto

**Comment by qudix** [comment]
No, some skse plugins are erroneously creating that folder and need to be updated by the author to fix it. BaseObjectSwapper is an example.

**Comment by ashemedai** [comment]
@z929669 Since this was confirmed to be an AddressLibrary/CommonLib issue, can this issue be closed?

Obligatory mention of [SKSE Log Redirector ](https://www.nexusmods.com/skyrimspecialedition/mods/144932) to work around the problem for now in case people stumble on this issue.
